corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -563,10 +563,21 @@ def get_username_in_last_form_user_id_submitted(domain, user_id):
         return user_submissions[0]['form']['meta'].get('username', None)
 
 
-def get_wrapped_ledger_values(domain, case_ids, section_id, entry_ids=None):
+def get_wrapped_ledger_values(domain, case_ids, section_id, entry_ids=None, pagination=None):
     # todo: figure out why this causes circular import
     from corehq.apps.reports.commtrack.util import StockLedgerValueWrapper
-    query = LedgerES().domain(domain).section(section_id).case(case_ids)
+    if pagination:
```

Comment:
This might be a little more clear if written:

```python
query = LedgerES().domain(domain).section(section_id).case(case_ids)
if pagination:
    query = query.size(pagination.count).start(pagination.start)
```

similar to how entry_ids is added to the query

Reply:
Done here: https://github.com/dimagi/commcare-hq/pull/20876/commits/6679fa056ecc97a1b95769373b696d88066dedd4